Parse/src/main/java/com/parse/ParsePushBroadcastReceiver.java

```diff
@@ -138,7 +138,9 @@ public void onReceive(Context context, Intent intent) {
   protected void onPushReceive(Context context, Intent intent) {
     JSONObject pushData = null;
     try {
-      pushData = new JSONObject(intent.getStringExtra(KEY_PUSH_DATA));
+      String pushDataStr = intent.getStringExtra(KEY_PUSH_DATA);
+      PLog.v(TAG, pushDataStr);
```

Comment:
Not sure whether this introduces too much overhead for production, but for debugging purpose, I think this is necessary since there is no reliable for us to tell whether a device has received a push notification or not.